src/fdtdx_studio/json_handling/Export.py

```
@@ -74,8 +82,9 @@ def build_material_list(self, list_obj):
 		"""builds the JSON serializable output of all Nonstandard Materials that were defined, calls build material for each"""
 		cfg = []
 		for obj in list_obj:
-			if obj[2]:
-				cfg.append(self.build_material(obj))
+			name, material, editable = obj
```

Ruff (0.15.10)

[warning] Unpacked variable `name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)